src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,13 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String palindrome = text.toLowerCase().replaceAll("[^a-z0-9]", "");
+        int length = palindrome.length();
+        String reversed = palindrome.substring(length / 2 + length % 2, length);
+        palindrome = palindrome.substring(0, length / 2);
```

Comment:
you don't need to divide it into two parts, just reverse your refactored string and compare it with its normal order